Android.bp

```diff
@@ -132,6 +132,9 @@ cc_library_shared {
         ":drm_hwcomposer_common",
         "bufferinfo/legacy/BufferInfoMinigbm.cpp",
     ],
+    cflags: [
+        "-DTEST_CALL_HWCOMPOSER_API",
+    ],
```

Comment:
can we add one more lib, name like hwc_evs.drm_minigbm.so? Because hwcomposer.drm_minigbm still need be used by  hwc service.
TEST_CALL_HWCOMPOSER_API can be renamed as EVS_HWCOMPOSER